spec/features/idv/steps/in_person/verify_info_spec.rb

```diff
@@ -25,7 +25,6 @@
     expect_in_person_step_indicator_current_step(t('step_indicator.flows.idv.verify_info'))
     expect(page).to have_current_path(idv_in_person_verify_info_path)
     expect(page).to have_content(t('headings.verify'))
-    expect(page).to have_current_path(idv_in_person_verify_info_path)
```

Comment:
Good eye! I wondered why this was being deleted. I ran it to ensure it was not causing a failure when it was present.  It is a dup from line 26. 👏 👏  Good clean up